reactor/shell_action.go

```diff
@@ -55,6 +55,7 @@ func (shact *ShellAction) formatCommand(message *ReactorDelivery) []string {
 			"{package}", message.GetPackageName(),
 			"{arch}", message.GetArch(),
 			"{repo}", message.GetRepoName(),
+			"{repository}", message.GetRepoName(),
```

Comment:
This is unnecessary, because this is just a interpolation variable. For example:

    This is a {repo} of the year!

...will replace `{repo}` with the actual name, like "This is a Groundhog Day of the year!". Now you just created an alias to that variable, so both `{repo}` and `{repository}` will interpolate. If you think that "repo" is not good, then remove it. Or remove "repository". In any case, we need to decide only for one.

Reply:
No, we should not decide for one.

Developer view: let's check the OBS docs for this notification stuff (https://openbuildservice.org/help/manuals/obs-admin-guide/obs.cha.administration.html#id-1.6.8.11.3.4.6)

...
__prefix__.package.build_success | A package build has succeeded | :repository, :arch, ...
__prefix__.repo.published | A repository was published | :project, :repo ...
...

Oh, ok, I will stick to the names as mentioned in the docs of the message producer. 

Oh, jenkobs does not work because it does react on "__prefix__.package.build_success" event but I don't get my value I want because the jenkobs starts mapping names implicitly and I am not aware of.

Reply:
But that is the thing: it has nothing to do with the OBS docs. The `repo` (or your proposed `repository`) are just variables inside the jenkobs. Those variables can be also called `pringles` or `christmas_tree` and still display values from the OBS. Think of "this variable will display me the repository name, comes from OBS". So you need to check jenkobs docs, not OBS's doc.

Reply:
Why do we only provide a limited amount of "replacer" at all? Why not simply accept any key and replace it with concrete value if available in the OBS JSON event?

Reply:
Yes, you are right and we can do that too. So your approach would be "Whatever comes from OBS just proxy-pass" (as long as you know what).

I have concern that this way would stick to the messy OBS's API and, second, will require to follow latest OBS documentation (correction: hacks, because the documentation is usually very outdated). If they add some more aliases like "packagesource" and/or "primary_repo", then our config will become even more complex. So I totally agree with you to get whatever they have (maybe even in a loop for "first win"), something like:

```go
for _, v := range []string{"repo", repository", "packagesource", "cosmic_ray_of_rpms", "othernonsense"} {
    name := rd.getContentString("repo")
    if name != "" {
        return name
    }
}
return ""
```

And so then the YAML would just refer to "repo", getting anything of these. What you think?